types/jquery-lazyload/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for jQuery Lazy Load 1.9
+// Project: https://github.com/tuupola/jquery_lazyload
+// Definitions by: Anderson Friaça <https://github.com/AndersonFriaca>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="jquery" />
+
+declare namespace jQueryLazyLoad {
```

Comment:
I know this is pedantic, but since `JQuery` starts with a capital `J`, can we be consistent here?

Reply:
There would be no inconsistencies, but I made the switch 😄